### PR TITLE
Add current and target fan state enums

### PR DESCRIPTION
**Pull request by Xeio**
Adding per https://github.com/home-assistant/core/pull/128618

### CODE REVIEW

File: aiohomekit/model/characteristics/const.py [hunk 1]
```diff
@@ -139,6 +139,23 @@ class SwingModeValues(enum.IntEnum):
     ENABLED = 1
 
 
+class CurrentFanState(enum.IntEnum):
+    """Possible values current fan state.
+    https://developer.apple.com/documentation/homekit/hmcharacteristicvaluecurrentfanstate"""
```
Comment (Jc2k) [inline aiohomekit/model/characteristics/const.py:144]:
```suggestion
    https://developer.apple.com/documentation/homekit/hmcharacteristicvaluecurrentfanstate
    """
```

File: aiohomekit/model/characteristics/const.py [hunk 2]
```diff
@@ -139,6 +139,23 @@ class SwingModeValues(enum.IntEnum):
...
+
+    INACTIVE = 0
+    IDLE = 1
+    ACTIVE = 2
+
+
+class TargetFanState(enum.IntEnum):
+    """Possible values target fan state.
+    https://developer.apple.com/documentation/homekit/hmcharacteristicvaluetargetfanstate"""
```
Comment (Jc2k) [inline aiohomekit/model/characteristics/const.py:153]:
```suggestion
    https://developer.apple.com/documentation/homekit/hmcharacteristicvaluetargetfanstate
    """
```

**Comment by Jc2k** [comment]
Cheers. Just had to fix a black nag. CI should push a new aiohomekit to pypi in a few minutes.

**Comment by Xeio** [comment]
Should I have also added this to \_\_init__.py? Just realized when I'm going to import it I have to use a different namespace

image

**Comment by Jc2k** [comment]
Oh good shout, let's do that.

**Comment by Xeio** [comment]
While I'm doing that... out of curiosity should I also have the word "Values" on the end? I sort of realized almost all the enums in that const.py have it on the name.

**Comment by Jc2k** [comment]
That sounds like something past me thought was a good idea, let's do that.

**Pull request merged**